yo/db.py

```diff
@@ -263,123 +283,81 @@ def _create_notification(self, table=None, **notification):
                                  notification)
             return False
 
-    def create_wwwpoll_notification(self, **notification):
```

Comment:
why not avoid repeating the same logic in `create_wwwpoll_notification` and `create_notification`? Why keep `_create_notification` otherwise?

Reply:
At some point i'll clean this up, for now it's of low priority - please create an issue for general refactoring of this type